.github/workflows/build.yml

```diff
@@ -268,15 +268,12 @@ jobs:
             ${{ github.workspace }}/packages/**/*.tgz
             ${{ github.workspace }}/packages/serverless/dist-awslambda-layer/*.zip
 
-  job_browserstack_test:
-    name: BrowserStack
+  job_browser_integration_tests:
+    name: Browser Integration Tests
     needs: job_build
     runs-on: ubuntu-latest
-    timeout-minutes: 45
+    timeout-minutes: 15
```

Comment:
Why 15?

Reply:
Excellent question.

- I first thought of removing this line.
- **45** was probably a historical artifact of how slow it was to run tests on BrowserStack (and the GHA worker was probably just idle waiting for a remote result).
- The [default is 360 minutes](https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_idtimeout-minutes), which is too much.
- **15** matches the other timeouts, and seems good enough to prevent this job from running too long before failing ([current run time is ~1 minute](https://github.com/rhcarvalho/sentry-javascript/runs/4407360132?check_suite_focus=true)).